sippy-ng/src/component_readiness/RegressedTestsModal.js

```diff
@@ -131,6 +131,34 @@ export default function RegressedTestsModal(props) {
       flex: 10,
       renderCell: (param) => <div className="test-name">{param.value}</div>,
     },
+    {
+      field: 'opened',
+      headerName: 'Regressed Since',
+      flex: 15,
+      valueGetter: (params) => params.row.regression_status.regressed_since,
+      valueGetter: (params) => {
```

Comment:
Is it intentional you have `valueGetter` defined twice here?

Reply:
Nope! TY, fixing now.

Reply:
I think most places in Sippy we're consistent about showing the relativeTime like "about 15 days ago" (there's a helper function), and use a `<Tooltip>` to show the raw value